test/tests/variables/lots_of_scalars/test.i

```diff
@@ -0,0 +1,26 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+[]
+
+[LotsOfVariables]
+  [./scalars]
+    number = 1000
+    family = SCALAR
+  [../]
+[]
+
+[Problem]
+  kernel_coverage_check = false
+[]
+
+[Preconditioning]
+  [./SMP]
+    type = SMP
+    full = true
+  [../]
+[]
+
+[Executioner]
+  type = Steady
+[]
```

Comment:
No `tests` file, so this is not executed as a part of `run_tests`.

Reply:
If you do that, you should probably restrict it like the other `lots_of_variables` test (`max_parallel=1`, no valgrind, no debug, etc) so that we don't get random fails

Reply:
BTW - Timing tests are notoriously terrible. We've dealt with the sporadic failure of the other "LotOf...." test for years. We really need to come up with a better way to test these things. How about we add a benchmark test instead? We do look at those....

Reply:
Well, given that the time went fro, >30s to <1s I think a generous max time of 2 should really be fine